### PR TITLE
Correct pyproject.toml to include the AEW API as a dependency.

### PR DESCRIPTION
**Pull request by phx-mkoninckx**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -34,11 +34,11 @@ dependencies = [
     "overrides>=6.1",
     "pyansys-tools-variableinterop>=0.1.1",
     "protobuf>=3.12.2",
+    "ansys-engineeringworkflow-api>=0.1.0",
 ]
 
 [project.optional-dependencies]
 doc=[
```
Comment (clatapie) [inline pyproject.toml:41]:
```suggestion
doc=[
    "ansys-engineeringworkflow-api==0.1.0",
```

File: pyproject.toml [hunk 2]
```diff
@@ -52,7 +52,6 @@ doc=[
     "sphinx-notfound-page==1.0.0",
 ]
 tests = [
```
Comment (clatapie) [inline pyproject.toml:54]:
```suggestion
tests = [
    "ansys-engineeringworkflow-api==0.1.0",
```

**Review by clatapie — CHANGES_REQUESTED** [review]
Using a specific `ansys-engineeringworkflow-api` version for tests and documentation build helps to ensure that a new package release of the latter doesn't cause any damage within `pymodelcenter` package.

**Review by phx-mpascale — APPROVED** [review]

**Review by clatapie — APPROVED** [review]
LGTM!

**Pull request merged**